src/pages/public_registration/action_accordions/DeletedRegistrationInformation.tsx

```diff
@@ -1,35 +1,79 @@
-import { Link, Skeleton, Typography } from '@mui/material';
+import { Box, Link, Skeleton, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
+import { fetchPerson } from '../../../api/cristinApi';
 import { fetchRegistration } from '../../../api/registrationApi';
-import { Registration, RegistrationStatus } from '../../../types/registration.types';
+import { ProfilePicture } from '../../../components/ProfilePicture';
+import { PublicationNote, Registration, RegistrationStatus } from '../../../types/registration.types';
+import { getIdentifierFromId } from '../../../utils/general-helpers';
+import { getFullCristinName } from '../../../utils/user-helpers';
 import { StyledStatusMessageBox } from '../../messages/components/PublishingRequestMessagesColumn';
 
 interface DeletedRegistrationInformationProps {
   registration: Registration;
+  unpublishingNote: PublicationNote;
 }
 
-export const DeletedRegistrationInformation = ({ registration }: DeletedRegistrationInformationProps) => {
+function extractId(createdBy: string) {
+  return '/cristin/person/' + createdBy.split('@')[0];
+}
+
+export const DeletedRegistrationInformation = ({
+  registration,
+  unpublishingNote,
+}: DeletedRegistrationInformationProps) => {
   const { t } = useTranslation();
+
+  const duplicateRegistrationIdentifier = getIdentifierFromId(registration.duplicateOf ?? '');
+
   const duplicateRegistrationQuery = useQuery({
-    enabled: !!registration.duplicateOf,
-    queryKey: ['registration', registration.duplicateOf],
-    queryFn: () => fetchRegistration(registration.duplicateOf ?? ''),
+    enabled: !!duplicateRegistrationIdentifier,
+    queryKey: ['registration', duplicateRegistrationIdentifier],
+    queryFn: () => fetchRegistration(duplicateRegistrationIdentifier),
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
+  const cristinIdentifier = unpublishingNote.createdBy ? extractId(unpublishingNote.createdBy) : null;
+  const personQuery = useQuery({
```

Comment:
Vi viser navn på avsender for alle supportmeldinger, hvor vi henter navn bare basert på username. Du kan se `MessageItem` for hvordan det er gjort der. Om de hentes likt vil vi slippe å hente navn på samme person flere ganger på samme side, siden responsen caches fra første oppslag. Om de hentes fra ulike steder kan man også risikere at de vises med ulikt navn på samme side (men det vil bare vil gjelde veldig få da)

Pro med å hente NVA-user (username) i stedet for Cristin Person:
- Frontend baserer seg på data (IDer) den får fra API, uten egen logikk for å bygge/gjette seg frem til ID på andre ressurser

Con med å hente NVA-user (username) i stedet for Cristin Person:
- Navn som ligger på NVA-user vil aldri oppdateres, så navn som ligger der kan være feil om person har endret navn

Blir pest eller kolera dette, så gjør det du føler er best :)